frontend/src/components/app/AppCard.vue

```diff
@@ -2,7 +2,10 @@
   <div class="card">
     <div class="card-image">
       <figure class="image is-animated is-5by3">
-        <img :src="app.imageUrl || 'https://bulma.io/assets/images/placeholders/1280x960.png'" alt="Placeholder image">
+        <img
+            :src="'http://localhost:8081/api/v1/tasks/' + app.namespace + '/' + app.version + '/logo.png'"
+            @error="($event.target as HTMLImageElement).src='https://bulma.io/assets/images/placeholders/1280x960.png'"
+            alt="Placeholder image">
```

Comment:
You should not hardcode the URL. What happen if the backend is at another url or port?
You should use a computed property instead  to provide the src image url and use `VITE_API_BASE_URL` variable instead for the localhost